Makefile

```diff
@@ -54,6 +54,10 @@ dist:
 .PHONY: dist-tools
 dist-tools: dist-csv-generator dist-csv-replace-imageref
 
+.PHONY: build-clean
+build-clean:
```

Comment:
do we want to join both clean operation to one call ?  or trigger is prior to build ?

Reply:
OK, better to remove this hunk from the patch and reevaluate later.